h2o-algos/src/main/java/hex/schemas/GLMV3.java

```diff
@@ -266,38 +300,45 @@ public static final class GLMParametersV3 extends ModelParametersSchemaV3<GLMPar
      * Only when balance_classes is enabled.
      * If not specified, they will be automatically computed to obtain class balance during training.
      */
-    @API(help = "Desired over/under-sampling ratios per class (in lexicographic order). If not specified, sampling factors will be automatically computed to obtain class balance during training. Requires balance_classes.", level = API.Level.expert, direction = API.Direction.INOUT)
+    @API(help = "Desired over/under-sampling ratios per class (in lexicographic order). If not specified, sampling" +
+            " factors will be automatically computed to obtain class balance during training. Requires balance_classes.",
+            level = API.Level.expert, direction = API.Direction.INOUT)
     public float[] class_sampling_factors;
 
     /**
      * When classes are balanced, limit the resulting dataset size to the
      * specified multiple of the original dataset size.
      */
-    @API(help = "Maximum relative size of the training data after balancing class counts (can be less than 1.0). Requires balance_classes.", /* dmin=1e-3, */ level = API.Level.expert, direction = API.Direction.INOUT)
+    @API(help = "Maximum relative size of the training data after balancing class counts (can be less than 1.0)." +
+            " Requires balance_classes.", /* dmin=1e-3, */ level = API.Level.expert, direction = API.Direction.INOUT)
     public float max_after_balance_size;
 
     /** For classification models, the maximum size (in terms of classes) of
      *  the confusion matrix for it to be printed. This option is meant to
      *  avoid printing extremely large confusion matrices.  */
-    @API(help = "[Deprecated] Maximum size (# classes) for confusion matrices to be printed in the Logs", level = API.Level.secondary, direction = API.Direction.INOUT)
+    @API(help = "[Deprecated] Maximum size (# classes) for confusion matrices to be printed in the Logs",
+            level = API.Level.secondary, direction = API.Direction.INOUT)
     public int max_confusion_matrix_size;
 
-    @API(help="Request p-values computation, p-values work only with IRLSM solver and no regularization", level = Level.secondary, direction = Direction.INPUT)
+    @API(help="Request p-values computation, p-values work only with IRLSM solver and no regularization",
+            level = Level.secondary, direction = Direction.INPUT)
     public boolean compute_p_values; // _remove_collinear_columns
 
     @API(help="If true, will fix tweedie variance power value to the value set in tweedie_variance_power.",
             level=Level.secondary, direction=Direction.INPUT)
     public boolean fix_tweedie_variance_power;
 
-    @API(help="In case of linearly dependent columns, remove some of the dependent columns", level = Level.secondary, direction = Direction.INPUT)
+    @API(help="In case of linearly dependent columns, remove some of the dependent columns", level = Level.secondary,
```

Comment:
```suggestion
    @API(help="In case of linearly dependent columns, remove some of the dependent columns", level = Level.secondary,
```
```suggestion
    @API(help="In case of linearly dependent columns, remove some of the dependent columns.", level = Level.secondary,
```

Reply:
Changed